app/client/src/components/ads/Dropdown.tsx

```diff
@@ -277,39 +272,47 @@ export const DropdownWrapper = styled.div<{
   isOpen: boolean;
 }>`
   width: ${(props) => props.width};
-  height: fit-content;
   z-index: 1;
-  background-color: ${(props) => props.theme.colors.dropdown.menu.bg};
-  border: 1px solid ${(props) => props.theme.colors.dropdown.menu.border};
-  padding: ${(props) => props.theme.spaces[3]}px 0;
+  background-color: ${Colors.WHITE};
   overflow: hidden;
+  overflow-y: auto;
+  box-shadow: 0px 12px 16px -4px rgba(0, 0, 0, 0.1),
+    0px 4px 6px -2px rgba(0, 0, 0, 0.05);
   display: ${(props) => (props.isOpen ? "inline-block" : "none")};
   .dropdown-search {
     margin: 4px 12px 8px;
     width: calc(100% - 24px);
 
     input {
-      height: 36px;
+      height: 32px;
       font-size: 14px !important;
-      color: ${Colors.GREY_10} !important;
+      color: ${Colors.GRAY_700} !important;
       padding-left: 36px !important;
+      border: 1.2px solid ${Colors.GRAY_200};
+
+      &:hover {
+        background: ${Colors.GRAY_50};
+      }
 
       &:focus {
-        border: 1.2px solid ${Colors.GREEN_1};
-        box-shadow: 0px 0px 0px 2px ${Colors.GREEN_2};
+        border-color: ${Colors.GRAY_900}!important;
```

Comment:
We can increase specificity by using multiple `&`. This should help us remove the need for `!important`
```suggestion
       &&&:focus { 
             border-color: ${Colors.GRAY_900}`,
```

Reply:
It doesn't work. That's why I added `!important`